src/app/components/Citations/APACitation.jsx

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import withCitationFields from './formatCitation';
+
+
+class APACitation extends React.Component {
+  componentDidMount() {
+    this.getComponentStrs();
+  }
+
+  getComponentStrs() {
+    this.authorsStr = this.formatAgentNames('authors');
+    // eslint-disable-next-line max-len
+    this.editorStr = `${this.formatAgentNames('editors', this.props.agents.authors.length > 0)}, (Ed${this.props.agents.editors.length === 1 ? '.' : 's.'})`;
```

Comment:
js stylistic thing, there's no need to postfix everything with Str, 

perhaps "formattedAuthors" to imply that stuff has been already transformed?  

Also, it's pretty useful to know if stuff is an array or not by the presence/absence of the pluralizer

Reply:
yeah, that's definitely better, was just trying to distinguish from the input and what the user would see.